100_essentials/coordinates/README.md

```diff
@@ -0,0 +1,33 @@
+# COORDINATES
```

Comment:
Just a quick note that I love everything about this challenge. ✨ 

This is a _super_ great way to teach variables to newcomers. I'm interested how hard the onboarding piece will be, but I think even if it's rough that's just an opportunity for documentation.